internal/usecase/sqldb/domains.go

```diff
@@ -147,7 +147,7 @@ func (r *DomainRepo) GetByName(_ context.Context, domainName, tenantID string) (
 			"tenant_id",
 		).
 		From("domains").
-		Where("name = ? AND tenant_id = ?", domainName, tenantID).
+		Where("LOWER(name) = LOWER(?) AND tenant_id = ?", domainName, tenantID).
```

Comment:
did you test this with both embedded and postgres?

Reply:
Yeah tested with both embedded and postgres.